VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: Javed Woof Telemetry


BO_ 1905 Javed: 7 Javed
 SG_ VoltageIn : 1|10@0+ (1,0) [0|0] "" Vector__XXX
 SG_ CurrentIn : 17|10@0+ (1,0) [0|0] "" Vector__XXX
 SG_ VoltageOut : 33|10@0+ (1,0) [0|0] "" Vector__XXX
 SG_ AmbientTemperature : 55|8@0+ (1,0) [0|0] "" Vector__XXX
 SG_ OverTemperature : 7|1@0+ (1,0) [0|0] "" Vector__XXX
 SG_ NoCharge : 6|1@0+ (1,0) [0|0] "" Vector__XXX
 SG_ UnderVoltage : 5|1@0+ (1,0) [0|0] "" Vector__XXX
 SG_ BatteryVoltageLevelReached : 4|1@0+ (1,0) [0|0] "" Vector__XXX

BO_ 1906 Woof: 7 Woof
 SG_ VoltageIn : 1|10@0+ (1,0) [0|0] "" Vector__XXX
 SG_ CurrentIn : 17|10@0+ (1,0) [0|0] "" Vector__XXX
 SG_ VoltageOut : 33|10@0+ (1,0) [0|0] "" Vector__XXX
 SG_ AmbientTemperature : 55|8@0+ (1,0) [0|0] "" Vector__XXX
 SG_ OverTemperature : 7|1@0+ (1,0) [0|0] "" Vector__XXX
 SG_ NoCharge : 6|1@0+ (1,0) [0|0] "" Vector__XXX
 SG_ UnderVoltage : 5|1@0+ (1,0) [0|0] "" Vector__XXX
 SG_ BatteryVoltageLevelReached : 4|1@0+ (1,0) [0|0] "" Vector__XXX

BO_ 1809 MpptPollJaved: 1 Telemetry
 SG_ Blank : 0|8@1- (1,0) [0|0] "" Vector__XXX

BO_ 1810 MpptPollWoof: 1 Telemetry
 SG_ Blank : 0|8@1- (1,0) [0|0] "" Vector__XXX




